File: perfect.js
let number = 0
for (let counter = 1; counter <= 1000; counter++) {
    let number = counter;
    let sum = 0;
    for (let count = 1; count < number; count++) {
        if (number % count === 0) {
            sum += count;
        }
    }
    if (number === sum) {
        console.log(sum);
    }
}

File: Perfect.java
public class Perfect{
    public static void main(String[] args){
            int number = 0;
            int sum = 0;

        for(int counter = 1; counter <= 1000; counter++){
             number = counter;
             sum = 0;
            for(int count= 1; count < number ; count ++){ 
                if(number % count == 0){
                sum += count;
                }
            }
              if(number == sum){
                 System.out.println(sum);
                }  
         }
        


}
}
